control-station/src/components/SensorBoxes/Camera.tsx

```diff
@@ -1,7 +1,38 @@
 import "./SensorData.css";
+import { LineChart } from "@mui/x-charts";
+import PodContext from "@/services/PodContext";
+import { useContext, useEffect, useState } from "react";
 
 function Camera() {
-	return <div className="camera"></div>;
+	const { podData } = useContext(PodContext);
+	const [lidarList, setLidarList] = useState<number[]>([]);
+
+	useEffect(() => {
+		console.log(lidarList);
+
+		if (lidarList.length < 200) {
+			setLidarList((prevLidarList) => [...prevLidarList, podData.lidar]);
+		} else {
+			setLidarList((prevLidarList) => {
+				const newList = [...prevLidarList.slice(1), podData.lidar];
+				return newList;
+			});
+		}
+	}, [podData.lidar, lidarList]);
```

Comment:
Issue: having `lidarList` in the dependency array will cause this hook to run infinitely.